style.css

```diff
@@ -242,7 +242,85 @@ button:hover {
 
 .aboutme-container {
   display: grid;
-  background-image: url('./img/abt-left.png'), url('./img/abt-bbgg.png');
+  background-image: url("./img/abt-left.png"), url("./img/abt-bbgg.png");
   background-position: left bottom, 120% 0%;
   background-repeat: no-repeat, no-repeat;
 }
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 64px 24px;
+}
```

Comment:
- [ ] Consider aligning the form to the center of the page so that it looks more professional and also matches that of your chosen design template from Figma 😄.

- [ ] Please add the image beside the form as you can see from the Figma design in the image I drop below.

Your design:

<img width="129" alt="chrome_W9rfqeXgpa" src="https://user-images.githubusercontent.com/81468675/144230676-d297f864-ec7c-4b58-b97d-1f6737ef66cf.png">

Figma design:

<img width="160" alt="chrome_aSHoGrd6kf" src="https://user-images.githubusercontent.com/81468675/144230831-0a019f56-5dee-4dd9-a8e0-cd54cfc2b523.png">

Reply:
Done 👍